docker-compose.yml

```diff
@@ -136,18 +109,22 @@ services:
       - AUTHENTICATOR_HOST=https://authenticator.levellingup.gov.localhost:4004
       - ACCOUNT_STORE_API_HOST=http://account-store:8080
       - APPLY_HOST=frontend.levellingup.gov.localhost:3008
+      - ASSESS_HOST=assessment.levellingup.gov.localhost:3010
       - FLASK_DEBUG=1
       - FLASK_ENV=development
       - REDIS_INSTANCE_URI=redis://redis-data:6379
       - SECRET_KEY=dc_key
+      - ASSESSMENT_STORE_API_HOST=http://pre-award-stores:8080/assessment
     env_file: .awslocal.env
     ports:
       - 3008:8080
+      - 3010:8080
```

Comment:
Ooh interesting. Any reason we're serving this up on a separate port where we didn't for the post-award consolidation?

Reply:
Mostly just so that people don't have to update all of their local URLs 😂

Reply:
Haha cool, that's a good enough reason to me 😄